### PR TITLE
[#1640] [Improvement] Standardize the invocation method for the Builder pattern 

### PR DESCRIPTION
**Pull request by charliecheng630**
### What changes were proposed in this pull request?

Standardize the invocation method for the Builder pattern 

### Why are the changes needed?

Make the code cleaner and more readable.

Fix: #1640 

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

The IT ( `./gradlew build` ) can ensure the function is work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,13 @@
 [![Last Committed](https://img.shields.io/github/last-commit/datastrato/gravitino)](https://github.com/datastrato/gravitino/commits/main/)
 [![OpenSSF Best Practices](https://www.bestpractices.dev/projects/8358/badge)](https://www.bestpractices.dev/projects/8358)
 
+
+[![License](https://img.shields.io/github/license/datastrato/gravitino)](https://github.com/datastrato/gravitino/blob/main/LICENSE)
+[![Contributors](https://img.shields.io/github/contributors/datastrato/gravitino)](https://github.com/datastrato/gravitino/graphs/contributors)
+[![Release](https://img.shields.io/github/v/release/datastrato/gravitino)](https://github.com/datastrato/gravitino/releases)
+[![Open Issues](https://img.shields.io/github/issues-raw/datastrato/gravitino)](https://github.com/datastrato/gravitino/issues)
+[![Last Committed](https://img.shields.io/github/last-commit/datastrato/gravitino)](https://github.com/datastrato/gravitino/commits/main/)
```
Comment (mchades) [inline README.md:22]: Why include these links? They appear to be duplicates of the ones above.

File: core/src/main/java/com/datastrato/gravitino/storage/relational/po/CatalogPO.java [hunk 2]
```diff
@@ -104,82 +104,91 @@ public int hashCode() {
...
 
     public CatalogPO.Builder withCatalogId(Long catalogId) {
-      metalakePO.catalogId = catalogId;
+      catalogPO.catalogId = catalogId;
       return this;
     }
 
     public CatalogPO.Builder withCatalogName(String name) {
-      metalakePO.catalogName = name;
+      catalogPO.catalogName = name;
```
Comment (mchades) [inline core/src/main/java/com/datastrato/gravitino/storage/relational/po/CatalogPO.java:119]: Could you please help to review the changes in this class? thanks! @yuqi1129

Comment (yuqi1129) [reply core/src/main/java/com/datastrato/gravitino/storage/relational/po/CatalogPO.java:119]: @mchades It's okay with me for the changes.

**Comment by jerryshao** [comment]
Do we cover all the `Builder` related codes in Gravitino?

**Comment by charliecheng630** [comment]
> Do we cover all the `Builder` related codes in Gravitino?

yes.

**Comment by mchades** [comment]
Good job! Overall LGTM, please resolve the conflicts. :)

**Comment by charliecheng630** [comment]
> Good job! Overall LGTM, please resolve the conflicts. :)

Already resolved. :)

**Comment by mchades** [comment]
> > Good job! Overall LGTM, please resolve the conflicts. :)
> 
> Already resolved. :)

Conflicts still exist:

image

**Comment by charliecheng630** [comment]
> > > Good job! Overall LGTM, please resolve the conflicts. :)
> > 
> > 
> > Already resolved. :)
> 
> Conflicts still exist:
> 
> image
> Maybe you need to update the main branch locally

Already resolved! Please review again, thank you.

**Review by mchades — APPROVED** [review]
LGTM, thanks for your hard work on this!

**Pull request merged**